.nyc_output/5c49856d-857e-44d9-b782-215413b42cb7.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Please add .nyc_output to our .gitignore file

Reply:
Thanks for your feedback. I just added .nyc_output to gitignore file